### ISSUE TITLE
feat [CLI]: endpoint to download the respective swagger collection and client code generation

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1689]
maybe something like:

`/meta/swagger.json`

or

`/meta/postman.json`

or make a convenient endpoint to generate it for different languages using swagger online code generator

`/meta/codegen/{language}`

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci minor`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.16.0](https://github.com/open-wa/wa-automate-nodejs/compare/3.15.2...3.16.0) (2021-05-19)

- build(deps): bump socket.io from 4.1.1 to 4.1.2 [`#1684`](https://github.com/open-wa/wa-automate-nodejs/pull/1684)
- build(deps): bump type-fest from 1.1.1 to 1.1.2 [`#1692`](https://github.com/open-wa/wa-automate-nodejs/pull/1692)
- build(deps): bump chrome-launcher from 0.13.4 to 0.14.0 [`#1693`](https://github.com/open-wa/wa-automate-nodejs/pull/1693)
- Release 3.16.0 [`da4d827`](https://github.com/open-wa/wa-automate-nodejs/commit/da4d8279193335e49995fe57972b1b34ee5d6e7b)
- 🔊 logs [CLI]: replace `console.log` with `Spin` to standardize CLI logs #1691 [`d08cdcd`](https://github.com/open-wa/wa-automate-nodejs/commit/d08cdcd56a02dcc8463b80b8a05516b76ce13772)
-   ✨ feat [CLI] : generate client libraries using `/meta/codegen` endpoint!! #1689 [`cb4c7b2`](https://github.com/open-wa/wa-automate-nodejs/commit/cb4c7b259e141217e972c9349cd497ff9d4b10aa)
- ✨ feat [CLI] : collection endpoints moved to `/meta/swagger.json` or `meta/postman.json` #1689 [`ef1fe20`](https://github.com/open-wa/wa-automate-nodejs/commit/ef1fe20af6011da27c4b39995326f5e114ea1ff7)
- ⏪ rollback: revert fix from #1670 - it broke api docs [`eb80142`](https://github.com/open-wa/wa-automate-nodejs/commit/eb80142989234af327646bc543276bd5d943943c)
- 🔒️ fixed some npm audit issues [`ee2c1c6`](https://github.com/open-wa/wa-automate-nodejs/commit/ee2c1c62ff8bd2ea1904fe2dc261e2fc7e471e4c)
- 🔧 config: explicitly remove some methods from collections generator [`d223c87`](https://github.com/open-wa/wa-automate-nodejs/commit/d223c87b26f1410277ff35d0c4ab98d0c4f8c41b)
- ✨ `Spin` is now publicly accessible [`0b28bc8`](https://github.com/open-wa/wa-automate-nodejs/commit/0b28bc8b3c2b96569de46465b766d8c1d41b5ac0)
- 🐛 fix: codegen results in proper method signatures #1690 [`faf3bac`](https://github.com/open-wa/wa-automate-nodejs/commit/faf3bacd0a425fd2ccdc54fa7893b9aa61c5395f)
